src/main/java/uk/gov/companieshouse/itemhandler/kafka/EmailSendMessageProducer.java

```diff
@@ -29,10 +29,14 @@ public EmailSendMessageProducer(final EmailSendMessageFactory avroSerializer, fi
      * @throws ExecutionException should something unexpected happen
      * @throws InterruptedException should something unexpected happen
      */
-    public void sendMessage(final EmailSend email)
+    public void sendMessage(final EmailSend email, String orderReference)
             throws SerializationException, ExecutionException, InterruptedException {
-        LOGGER.trace("Sending message to kafka producer");
-        final Message message = emailSendAvroSerializer.createMessage(email);
-        emailSendKafkaProducer.sendMessage(message);
+        Map<String, Object> logMap = LoggingUtils.createLogMap();
+        LoggingUtils.logIfNotNull(logMap, LoggingUtils.ORDER_REFERENCE_NUMBER, orderReference);
+        LOGGER.info("Sending message to kafka producer", logMap);
```

Comment:
* We see this pattern a lot: 1) create map 2) populate map with key-value pairs 3) log message with map.
* It might be good to introduce convenience methods in `LoggingUtils` for example that can be used to do this in a single call?

Reply:
I'll do this for common usages of logging e.g. order reference number